google/cloud/security/inventory/inventory_loader.py

```diff
@@ -234,14 +225,16 @@ def _send_email(organization_id, cycle_time, cycle_timestamp, status, pipelines,
     except EmailSendError:
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
+# TODO: Investigate improving so the pylint disable isn't needed.
+# pylint: disable=too-many-locals
 def main(argv):
```

Comment:
Adding a TODO here, and will follow up in a separate PR, with the proper testing.